content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -44,7 +47,7 @@ secretGenerator:
     The `-n` flag ensures that there's no newline character at the end of your
     files.
 
-1.  Create the `kustomization.yaml` file:
+2.  Create the `kustomization.yaml` file:
```

Comment:
don't change this.

Reply:
👋 @duddu .
Do you have time to continue with this pull request?